.github/workflows/ci.yml

```diff
@@ -48,13 +48,14 @@ jobs:
   test:
     name: test ${{ matrix.os-name }}, v${{ matrix.node-version }}
     runs-on: ${{ matrix.os }}
+    needs: [lint]
```

Comment:
We do it for matrix-hosted at least, because it's a cost saver. I think there are a few other places too. Granted, it's not required for the change here so I can revert.